lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendar.m

```diff
@@ -67,9 +67,10 @@ - (void)setupCalendar {
     self.utcCalendar.timeZone = [[NSTimeZone alloc] initWithName:@"UTC"];
 }
 
-- (void)setRct_selectionType:(BPKCalendarSelection)rct_selectionType {
-    if(_rct_selectionType != rct_selectionType) {
-        _rct_selectionType = rct_selectionType;
+- (void)setRct_selectionConfiguration:(BPKCalendarSelectionConfiguration *)rct_selectionConfiguration {
+    // Object comparison will always be false as we're using new instances each time?
```

Comment:
Worth verifying this

Reply:
This was actually a note to myself to remind myself to remove it 😆